src/Request/SearchRequest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace DDB\OpenPlatform\Request;
+
+use DDB\OpenPlatform\Response\SearchResponse;
+
+/**
+ * Perform a search query.
+ *
+ * @property array $fields
+ *   Fields to request per material.
+ * @property string $query
+ *   The query to run. Required.
+ * @property int $offset
+ *   Starting offset in result.
+ * @property int $limit
+ *   Maximum number of results returned.
+ * @property string $sort
+ *   Sort order.
+ * @property string $profile
+ *   Search profile.
+ *
+ * For further information, see the /search call at
+ * https://openplatform.dbc.dk/v3/
```

Comment:
It would be good with a direct link to the search related part of the service e.g. https://openplatform.dbc.dk/v3/#operations-tag-Search or https://openplatform.dbc.dk/v3/#operations-Search-get_search (GET has more detail than POST).